main.ts

```diff
@@ -20,10 +21,12 @@ import linkCommand from "@/subcommands/link/mod.ts";
 
 const DEFAULT_LOG_LEVEL = "INFO";
 const { logLevel = DEFAULT_LOG_LEVEL } = await getConfig().get();
+const { version } = JSON.parse(await Deno.readTextFile("./deno.jsonc"));
```

Comment:
这样看行不
https://v8.dev/features/import-assertions
https://github.com/tc39/proposal-import-attributes
```suggestion
import { version } from "./deno.jsonc" assert { type: 'json' };
```

Reply:
<img width="938" alt="Screenshot 2024-04-30 at 10 03 42" src="https://github.com/Byzanteam/jcli/assets/91400014/3fbe7b38-90c9-4a0a-afb4-4ede9307631d">

Reply:
感觉不太行，这个我昨天测试过了，只有 `deno.json` 可以 `deno.jsonc` 不行。

Reply:
你把 jsonc 改成 json 吧，反正我们也没用到 comment